tests/cypress/e2e/features/analytics_pipeline.js

```diff
@@ -194,35 +186,17 @@ context('Analytics pipeline', () => {
             cy.get('#cvat_canvas_shape_2').should('not.exist');
             cy.saveJob();
 
-            cy.logout();
-            cy.getAuthKey().then((res) => {
-                const authKey = res.body.key;
-                cy.request({
-                    method: 'POST',
-                    url: 'api/analytics/reports',
-                    headers: {
-                        Authorization: `Token ${authKey}`,
-                    },
-                    body: {
-                        project_id: projectID,
-                    },
-                }).then((response) => {
-                    const rqID = response.body.rq_id;
-                    waitForReport(authKey, rqID);
-                });
-            });
-            cy.login();
-            cy.intercept('GET', '/api/analytics/reports**').as('getReport');
-
             cy.visit(`/projects/${projectID}`);
             openAnalytics('project');
-            cy.wait('@getReport');
+            cy.get('.cvat-empty-performance-analytics-item').should('exist').and('be.visible').within(() => {
+                cy.get('button').contains('Request').click();
+            });
+
```

Comment:
Why dont we wait here for report request?  It can improve test stability

Reply:
How will it impove test stability? 

`should('exist')` also will wait up to default timeout.

Reply:
On the second tought yes, default `sould` timeout should be enough